### ISSUE TITLE
Deploy Site to free web server

### ISSUE THREAD

**Issue by ansarimofid** [devists/projectile#17]
You can use any of the below to deploy
- [PythonAnywhere](https://www.pythonanywhere.com/)
- [OpenShift](https://www.openshift.com)
- [heroku](https://www.heroku.com/)

**Comment by EmilLuta** [comment]
@ansarimofid Is this still available?

**Comment by ansarimofid** [comment]
Yes

**Comment by roboflank** [comment]
@ansarimofid can you include the requirements.txt file

**Comment by EmilLuta** [comment]
@ansarimofid I totally agree that you lack setup process. I'll give you a hand of help on that one. As soon as I figure it out.

**Comment by EmilLuta** [comment]
@ansarimofid I have created a pull request you might want to look into + deployed it to pythonanywhere.

@denzelwamburu If my PR gets accepted, you'll have the data you need.

**Comment by ansarimofid** [comment]
any progress?

**Comment by jaykishan007** [comment]
yeah... Search is about to complete.. and hope you guys receive a pull request by tomorrow.

**Comment by anshuldawar** [comment]
I have created the http://projectile.pythonanywhere.com/ for the following 3 months.

If you need help with deployment, let me know.
